mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
+    if (!deletionResult) {
+      return res.status(404).json({ error: "Order not found" });
```

Comment:
The error response message should be consistent throughout the codebase. In the `exports.create` function, the error response message is `error: errorHandler(error)`, but in this snippet, it's `error: "Order not found"`. For consistency, it should be `error: errorHandler("Order not found")`.